### DISCUSSION TITLE
pmtiles convert : Anyway to specificy temp dir and to resume from aborted conversion ?

### DISCUSSION THREAD

**Discussion by FabriceACohen** [protomaps/PMTiles#229] [Q&A]
I started a pmtiles convert from planet : it takes a lot of time... and when I woke up I felt on "no more space on disk".
1 - I didn't know it was using my tmp drive
2 - After a dozen of hours I wish I could resume the process.

Thanks.

**Comment by bdon** [comment]
are you using this to convert, or python? https://github.com/protomaps/go-pmtiles

A dozen hours sounds strange, how large is your tileset and what format is it in?

**Reply by FabriceACohen** [reply]
Yes I use this tool for windows. 

I am using a planet.mbtiles file which is around 70Go

**Reply by bdon** [reply]
how much RAM do you have on your desktop and what are the specs? it shouldn't take that long, unless you started swapping to disk.
